tendrl-dashboard.spec

```diff
@@ -9,6 +9,7 @@ Source0: %{name}-%{version}.tar.gz
 URL: https://github.com/Tendrl/dashboard
 
 Requires: tendrl-api
```

Comment:
tendrl-frontend is not using any api from tendrl-api. However tendrl-frontend needs tendrl-api-httpd package which does some common configuration for both api and tendrl-frontend.